### ISSUE TITLE
Version-Sensitive Solaar Log Not Mentioned by README

### ISSUE THREAD

**Issue by Easy121** [marcelhoffs/input-switcher#13]
Finally get this working after an afternoon's work ... The result is great. Now I can switch both keyboard and mouse between a windows 11 laptop and an Ubuntu 20.04 desktop computer instantly. 

### Main Hindrance
When I found that `switch_to_2.bat` does not work, turned to `6 - Modify the scripts` in README, and tested in Ubuntu with `solaar -dd config "MX Keys" change-host 1`, there is nothing like this:
```
logitech_receiver.base: (18) <= w[11 01 091E 00000000000000000000000000000000]
```

### Reason
The version of Solaar I used for testing is 1.1.8, which is too new for `w[11 01 091E ...` to output. The debug output of Solaar has been frequently modified in recent versions (modifications found in 1.1.6, 1.1.4, 1.1.2). I tried Solaar 1.1.1 and obtained the desired output. 
![image](https://user-images.githubusercontent.com/68587713/229285399-23c2479f-0d81-43d8-9836-167cacf5bb7d.png)

Few things to note:

1. To use Solaar 1.1.1, download [here](https://github.com/pwr-Solaar/Solaar/releases/tag/1.1.1) and run `bin/solaar -dd config "MX Keys" change-host 1` in its root directory. If error like `Solaar: missing required package 'python-pyudev'` is witnessed, you can consult [this](https://github.com/pwr-Solaar/Solaar/issues/782#issuecomment-1492918992). 
2. If you want to test `switch_to_2.bat` in win11, double-click it in `C:\Program Files\InputSwitcher` instead of in other directory. And also don't use `run as an administrator`. Running in other directory or as an administrator won't work in my experience. 

### REAME Update
In conclusion, the README might be updated to inform the requirement of Solaar 1.1.1 to output the command we need.

**Comment by Id3aFly** [comment]
Thank you! That helped. Can you propose a merge request for the readme for this?

**Comment by jesssicachen** [comment]
Thank you so much! you are my life saver

**Comment by shbatm** [comment]
You just need more `d`! using the following should get you the verbose debugging output:

```
solaar -ddd config "MX Keys" change-host 1
```
